account_due_list_aging_comment/README.rst

```diff
@@ -0,0 +1,63 @@
+===============================
+Payments Due list aging comment
+===============================
+This module extends the Payments and Due list to add the capability to introduce aging comments in-line.
+
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+
+
+Usage
+=====
+
+To use this module, you need to go to:
+
+Accounting -> Adviser -> Payments and due list
```

Comment:
Invoicing -> Accounting -> Payments and due list

Reply:
Done and also is squash in only one commit. Thanks for your revision